source/include/azure_iot_adu_client.h

```diff
@@ -387,15 +393,15 @@ AzureIoTResult_t AzureIoTADUClient_SendResponse( AzureIoTADUClient_t * pxAzureIo
  *
  * @param[in] pxAzureIoTADUClient The #AzureIoTADUClient_t * to use for this call.
  * @param[in] pxAzureIoTHubClient The #AzureIoTHubClient_t * to use for this call.
- * @param pxDeviceProperties The device information which will be used to generate the payload.
- * @param pxAduUpdateRequest The current #AzureIoTADUUpdateRequest_t. This can be `NULL` if there isn't currently
+ * @param[in] pxDeviceProperties The device information which will be used to generate the payload.
+ * @param[in] pxAduUpdateRequest The current #AzureIoTADUUpdateRequest_t. This can be `NULL` if there isn't currently
  * an update request.
- * @param xAgentState The current #AzureIoTADUAgentState_t.
- * @param pxUpdateResults The current #AzureIoTADUClientInstallResult_t. This can be `NULL` if there aren't any
+ * @param[in] xAgentState The current #AzureIoTADUAgentState_t.
+ * @param[in] pxUpdateResults The current #AzureIoTADUClientInstallResult_t. This can be `NULL` if there aren't any
  * results from an update.
- * @param pucBuffer The buffer into which the generated payload will be placed.
- * @param ulBufferSize The length of \p pucBuffer.
- * @param pulRequestId An optional request id to be used for the publish. This can be `NULL`.
+ * @param[in] pucBuffer The buffer into which the generated payload will be placed.

```

Comment:
Should this one be out?

Reply:
Yea I guess we can. It isn't to be used by the user after this function returns since the API sends it. But it's more of an out than passing data in you're right.

Reply:
There were a few others that seemed to have a similar situation as well from the wording in the comment

Reply:
I went through http and crypto as well to do the same thing. Thanks for the callout!